cmd/provisioner-localpv/app/helper_hostpath.go

```diff
@@ -50,8 +50,9 @@ var (
 	//to be launched.
 	CmdTimeoutCounts = 120
 
-	//conversion stores the equivalent of 1kb stored as 1kb, 1mb, 1gb, 1tb
-	conversion = []float64{1, 1024, 1048576, 1073741824}
+	//conversion stores the size as 1kb, 1mb, 1gb, 1tb
+	//respectively in kb
+	conversion = []float64{math.Pow(2, 0), math.Pow(2, 10), math.Pow(2, 20), math.Pow(2, 30)}
```

Comment:
This is not used anywhere? Is this array required?

Reply:
No, I will remove it . It was used previously